### ISSUE TITLE
Should rescue StandardError instead of RuntimeError to make failed_auto_loading_is_not_found useful?

### ISSUE THREAD

**Issue by pkmiec** [stffn/declarative_authorization#94]
In load_object, there is a begin / rescue around the find. I believe the intent is to catch errors produced by find. However, AR's find throws StandardError's (like RecordNotFound). So I think the code should read as follows:

```
begin
  object = load_object_model.find(contr.params[:id])
rescue => e
```

??

**Comment by stffn** [comment]
Fixes failed_auto_loading_is_not_found, closed by 274170c3a1abc081af9ab315cce64c1014548774

**Comment by stffn** [comment]
Thanks for the hint
